### PR TITLE
CB-3873 SDX to configure HMS with the required changes for password b…

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp [hunk 1]
```diff
@@ -219,6 +219,12 @@
                         "roleType": "HIVEMETASTORE"
                     }
                 ],
+                "serviceConfigs": [
+                    {
+                        "name": "hive_metastore_enable_ldap_auth",
```
Comment (doktoric) [inline core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: should it be in the configprovider?

Comment (topolyai5) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: no, it should not, it is fixed for sdx and could not be set by user

Comment (doktoric) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: we are setting hive_metastore_ldap_uri in the configprovider which means every cluster will use this property

Comment (topolyai5) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: Yes, but we don't enable the feature.

Comment (topolyai5) [reply core/src/main/resources/defaults/blueprints/cdp-sdx-ha.bp]: we don't know is it a DL or not, here: https://github.com/hortonworks/cloudbreak/pull/6526/files#diff-3ba4d7ee120408de64e69ad4097ed9da

**Pull request merged**